python/lsst/coadd/utils/coaddDataIdContainer.py

```diff
@@ -72,3 +71,45 @@ class ExistingCoaddDataIdContainer(CoaddDataIdContainer):
     def makeDataRefList(self, namespace):
         super(ExistingCoaddDataIdContainer, self).makeDataRefList(namespace)
         self.refList = [ref for ref in self.refList if ref.datasetExists()]
+
+class TractDataIdContainer(CoaddDataIdContainer):
```

Comment:
Do we need this? It's not used anywhere in the stack and it comes with no tests, so it seems like we're just adding debt with no concrete gains.